packages/pieces/community/interfaces/src/lib/triggers/interface-trigger.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  Property,
+  TriggerStrategy,
+  createTrigger,
+} from '@activepieces/pieces-framework';
+
+export const interfaceTrigger = createTrigger({
+  name: 'interface_trigger',
+  displayName: 'Interface',
+  description: 'Trigger the flow through an interface.',
+  props: {
+    textInputs: Property.Array({
```

Comment:
This is not an extensible structure. For example, when we add a new file type, we end up adding multiple arrays, and we also impose an order on the interface.

I would suggest renaming this to 'inputs,' removing 'placeholder' from the text field, and adding a static dropdown field called 'Type' with "FILE" and "TEXT" as options.

This will pose a challenge in customizing each object inside the array later, depending on the type. However, I believe this is where we should focus on solving in the future, rather than creating multiple arrays.